cloudpickle/cloudpickle.py

```diff
@@ -307,6 +328,8 @@ def save_function_tuple(self, func):
         save(_fill_function)  # skeleton function updater
         write(pickle.MARK)    # beginning of tuple that _fill_function expects
 
+        self._save_subimports(code, set(f_globals.values()) | set(closure))
```

Comment:
This line creates a regression: https://github.com/cloudpipe/cloudpickle/issues/86